packages/editor/src/store/test/selectors.js

```diff
@@ -1302,7 +1263,16 @@ describe( 'selectors', () => {
 	} );
 
 	describe( 'isEditedPostAutosaveable', () => {
+		// isEditedPostAutosaveable is only implemented as a registry selector
+		// to handle a deprecation. None of the tests here trigger the deprecation,
```

Comment:
"None of the tests here trigger the deprecation" reads like a lack of needed coverage. Though, depending on your response to my prior comment about avoiding the deprecation, it may not be necessary.